JavaErickHernandez/Day2/JavaFormatting.java

```diff
@@ -0,0 +1,17 @@
+package bootcamp;
+
+import java.util.Scanner;
+
+public class JavaFormatting {
+	public static void Main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        System.out.println("================================");
+        for(int i=0;i<3;i++){
+            String s1=sc.next();
+            int x=sc.nextInt();
+            System.out.printf("%-15s%03d\n", s1,x);
+        }
+        System.out.println("================================");
```

Comment:
Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings